### PR TITLE
LPS-81405 Test

### PR DESCRIPTION
**Pull request by brianchiu**

### CODE REVIEW

**Comment by brianchiu** [comment]
ci:test:relevant

**Comment by brianchiu** [comment]
ci:stop

**Comment by brianchiu** [comment]
ci:test:relevant

**Comment by brianchiu** [comment]
ci:test

File: portal-impl/src/content/Language.properties [hunk 1]
```diff
@@ -2466,7 +2466,8 @@ layout.types.embedded=Embedded
 layout.types.embedded.description=Show content from another website.
 layout.types.panel=Panel
 layout.types.panel.description=Create a page with predefined applications and navigation.
-layout.types.portlet=Layout
+layout.types.portlet=Widget Page
+layout.types.portlet.description=Build a page from a layout and adding widgets and content.
```
Comment (anthony-chu) [inline portal-impl/src/content/Language.properties]: @brianchiu i think this one needs to be corrected as well - 

as i read it, two ways of understanding the phrase show up:

1. Build a page from a layout, adding widgets and content
1. Build a page from a layout and add widgets and content

The verb tenses don't match in the way that it's currently used.

**Comment by brianchiu** [comment]
ci:test:relevant

**Comment by brianchiu** [comment]
Testray results here:
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=329029574

**Pull request closed without merging**